src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -101,7 +107,18 @@ public CheckInProcessRecords withHighestPriorityFulfillableRequest(Request reque
       this.loan,
       this.requestQueue,
       this.checkInServicePoint,
-      request);
+      request, loggedInUserId);
```

Comment:
Similar to other comments, please either format the parameters up to end of a line or as a parameter per line, rather than a mixture.

Reply:
Done.